packages/code-du-travail-frontend/pages/_document.js

```diff
@@ -42,27 +42,17 @@ export default class MyDocument extends Document {
     return (
       <html lang="fr" prefix="og: http://ogp.me/ns#">
         <Head>
-          <link rel="stylesheet" type="text/css" href="/static/fonts.css" />
+          <link
+            href="https://fonts.googleapis.com/css?family=Merriweather:400,700|Open+Sans:400,600i&display=swap"
```

Comment:
I think we should host our fonts (poke @revolunet )

Reply:
yes we have to :)

Reply:
We can inline them as they were previously (like it was before). But it is more convenient like that for now. Inline the fonts increases the bundle size but remove the "font swap" you can see during the first load.
I have a preference for the "swap" but it is visually less elegant than inlining the fonts

Reply:
yes we have to host them or embed them directly :/

Reply:
we probably want to subset them to :)